src/main/java/org/folio/circulation/domain/ChangeDueDateRepository.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.circulation.domain;
+
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.ServerErrorFailure;
+
+import java.lang.invoke.MethodHandles;
+import java.util.concurrent.CompletableFuture;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ChangeDueDateRepository {
```

Comment:
What is a change due date repository?

I typically think of a repository as relating to an entity, this class appears to relate to a very specific operation. What extra information does the suffix `repository` add to the name?

Reply:
It more of a service in fact. I will rename it.